### ISSUE TITLE
Runtime: CRTs - 2nd class shares/tokens that do not get revenue share for creator

### ISSUE THREAD

**Issue by mochet** [Joystream/joystream#5153]
# Problem
Described here: https://github.com/Joystream/atlas/issues/6340
TLDR: The $CRT's owners tokens mean they get a much higher proportion of the revenue share, which makes CRTs less attractive to investors.

# Solution
There may be other ways to solve this but it could be that there is some way to mark the creators tokens, or a proportion of them as being incapable of participating in a revenue share--this would mean the creator can still sell their tokens on the market but not get an extreme advantage when revenue shares are started. This kind of adds some balance so the creator is profiting more from their CRT having liquidity vs being able to just take huge amounts of the revenue share cut.

Due to CRTs already having been launched it would be ideal for a creator to be able to change some % of their current CRT holdings into non-revenue share tokens.
